test/test.properties

```diff
@@ -0,0 +1,21 @@
+test.server.name=ce-dev5.gluu.org
```

Comment:
if we want to keep it for `ce-dev5.gluu.org` then rename this file to `ce-dev5.properties`

Reply:
Not Required, so removed.